nix/default.nix

```diff
@@ -12,22 +12,29 @@ import sources.nixpkgs {
     })
     (import "${sources.gomod2nix}/overlay.nix")
     (_: pkgs: {
-      hermes = pkgs.rustPlatform.buildRustPackage rec {
+      hermes = pkgs.stdenv.mkDerivation rec {
+        version = "v1.0.0-rc.2";
         name = "hermes";
-        src = sources.ibc-rs;
-        cargoSha256 = sha256:1sc4m4bshnjv021ic82c3m36pakf15xr5cw0dsrcjzs8pv3nq9cd;
-        cargoBuildFlags = "-p ibc-relayer-cli";
-        buildInputs = pkgs.lib.optionals pkgs.stdenv.isDarwin [
-          pkgs.darwin.apple_sdk.frameworks.Security
-          pkgs.darwin.libiconv
-        ];
-        doCheck = false;
-        RUSTFLAGS = "--cfg ossl111 --cfg ossl110 --cfg ossl101";
-        OPENSSL_NO_VENDOR = "1";
-        OPENSSL_DIR = pkgs.symlinkJoin {
-          name = "openssl";
-          paths = with pkgs.openssl; [ out dev ];
-        };
+        src = pkgs.fetchurl {
```

Comment:
it was better to put the build expression in a separate file since it's big enough now, and isn't it more convenient to build from source, although they need a longer build timer for the first time.